AirSkyBoat/AirSkyBoat
# Fix issue with item add option and add effects

**_I affirm:_**


## Please enter a player-facing description
The winds that drop from sky gods now properly remove the added effects of corresponding gods. (Tracent)
Many weapons with added effects for certain mob families now work correctly. (Tracent)

## What does this pull request do? (Please be technical)
This PR fixes an issue caused by a mismatch between the ID numbers of ITEM_ADDEFFECT_OPTION mod and the actual numbers being used in the SQL. Also fixes an issue whereby certain added effects on ranged weapons (such as god winds) would have the chance of procing incorrectly lowered.

## Steps to test these changes
Test a god wind on the corresponding god to remove the added effect. Also use A l'Outrance for example on beasts to see added effect damage.

## Special Deployment Considerations
